tests/test_hubspot_newfw_all_fields.py

```diff
@@ -0,0 +1,312 @@
+        'contactCounts',
+    },
+    'owners': {  # BUG https://jira.talendforge.org/browse/TDL-15000
+        'activeSalesforceId'
+    },
+    'forms': {  # BUG https://jira.talendforge.org/browse/TDL-15001
+        'alwaysCreateNewCompany',
+        'themeColor',
+        'publishAt',
+        'editVersion',
+        'embedVersion',
+        'themeName',
+        'style',
+        'thankYouMessageJson',
+        'createMarketableContact',
+        'kickbackEmailWorkflowId',
+        'businessUnitId',
+        'portableKey',
+        'parentId',
+        'kickbackEmailsJson',
+        'unpublishAt',
+        'internalUpdatedAt',
+        'multivariateTest',
+        'publishedAt',
+        'customUid',
+        'isPublished',
+        'paymentSessionTemplateIds',
+        'selectedExternalOptions',
+    },
+    'companies': {  # BUG https://jira.talendforge.org/browse/TDL-15003
+        'mergeAudits',
+        'stateChanges',
+        'isDeleted',
+        'additionalDomains',
+        'property_hs_analytics_latest_source',
+        'property_hs_analytics_latest_source_data_2',
+        'property_hs_analytics_latest_source_data_1',
+        'property_hs_analytics_latest_source_timestamp',
+    },
+    'campaigns': {  # BUG https://jira.talendforge.org/browse/TDL-15003
+        'lastProcessingStateChangeAt',
+        'lastProcessingFinishedAt',
+        'processingState',
+        'lastProcessingStartedAt',
+    },
+    'deals': {  # BUG https://jira.talendforge.org/browse/TDL-14999
+        'imports',
+        'property_hs_num_associated_deal_splits',
+        'property_hs_is_deal_split',
+        'stateChanges',
+        'property_hs_num_associated_active_deal_registrations',
+        'property_hs_num_associated_deal_registrations',
+        'property_hs_analytics_latest_source',
+        'property_hs_analytics_latest_source_timestamp_contact',
+        'property_hs_analytics_latest_source_data_1_contact',
+        'property_hs_analytics_latest_source_timestamp',
+        'property_hs_analytics_latest_source_data_1',
+        'property_hs_analytics_latest_source_contact',
+        'property_hs_analytics_latest_source_company',
+        'property_hs_analytics_latest_source_data_1_company',
+        'property_hs_analytics_latest_source_data_2_company',
+        'property_hs_analytics_latest_source_data_2',
+        'property_hs_analytics_latest_source_data_2_contact',
+    },
+    'subscription_changes':{
+        'normalizedEmailId'
+    }
+}
+
+class HubspotAllFieldsTest(AllFieldsTest, HubspotBaseCase):
+    """Hubspot all fields test implementation """
+
+    @staticmethod
+    def name():
+        return "tt_hubspot_all_fields"
+
+    def streams_to_test(self):
+        """expected streams minus the streams not under test"""
+        return self.expected_streams().difference({
+            'owners',
+            'subscription_changes', # BUG_TDL-14938 https://jira.talendforge.org/browse/TDL-14938
+        })
+
+    def setUp(self):
```

Comment:
Usually setUp is written to only run once even if multiple tests are called.  This implementation seems to only have one test so it may not be an issue but I'd like to maintain the pattern if possible.

Reply:
Let's discuss on how to do this tap specific setUp